mbuild/formats/lammpsdata.py

```diff
@@ -181,33 +196,37 @@ def write_lammpsdata(structure, filename, atom_style='full'):
             sigmas = [atom.sigma for atom in structure.atoms]
             epsilon_dict = dict([(unique_types.index(atom_type)+1,epsilon) for atom_type,epsilon in zip(types,epsilons)])
             sigma_dict = dict([(unique_types.index(atom_type)+1,sigma) for atom_type,sigma in zip(types,sigmas)])
-            data.write('\nPair Coeffs # lj\n\n')
+            data.write('\nPair Coeffs # lj \n\n')
+            data.write('#\tepsilon sigma \t     (kcal/mol, \tAngstrom)\n')
```

Comment:
```suggestion
            data.write('#\tepsilon sigma \t\n(kcal/mol, \tAngstrom)\n')
```
This sort of formatting, on this comment and others, would be a little more legible